src/plugins/foxPage/components/AssetActions.tsx

```diff
@@ -36,6 +37,7 @@ type FoxTabProps = {
 
 const GetFoxyModalRoute = `/defi/token_staking/ShapeShift/overview`
 const GetFoxCoinbaseExternalUrl = `https://www.coinbase.com/price/fox-token`
+const TradeFoxyElasticSwapUrl = `https://elasticswap.org/`
```

Comment:
Since ElasticSwap doesn't have routing implemented for the current pair as queryparams, we can use `https://elasticswap.org/#/swap` at least, which is one less click for users

Reply:
Fixed !